tests/CarbonInterval/ExpressiveComparisonTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+declare(strict_types = 1);
+
+/**
+ * This file is part of the Carbon package.
+ *
+ * (c) Brian Nesbitt <brian@nesbot.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace Tests\CarbonInterval;
+
+use Carbon\CarbonInterval;
+use Tests\AbstractTestCase;
+
+class ExpressiveComparisonTest extends AbstractTestCase
```

Comment:
We will have later ->greaterThen, ->lowerThan, etc. it will be a lot of files have 1 class for each method. I would prefer to have all the tests in `ComparisonTest`

Still each method can have 1 test method.